go.mod

```diff
@@ -104,28 +105,27 @@ require (
 	github.com/opencontainers/runc v1.0.2
 	github.com/opencontainers/selinux v1.8.2
 	github.com/otiai10/copy v1.6.0
-	github.com/pierrec/lz4 v2.6.0+incompatible
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/dynamiclistener v0.2.7
 	github.com/rancher/lasso v0.0.0-20210616224652-fc3ebd901c08
 	github.com/rancher/remotedialer v0.2.0
-	github.com/rancher/wharfie v0.3.4
+	github.com/rancher/wharfie v0.5.1
 	github.com/rancher/wrangler v0.8.8
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/rootless-containers/rootlesskit v0.14.5
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/tchap/go-patricia v2.3.0+incompatible // indirect
-	github.com/urfave/cli v1.22.2
+	github.com/urfave/cli v1.22.4
```

Comment:
Haven't we been sticking to urfave/cli v1.22.2 because of a regression with string-slice args?

Reply:
if so we never pinned it with a `replace` entry.